src/Data/Text/Utf16/Rope/Mixed.hs

```diff
@@ -38,6 +38,11 @@ module Data.Text.Utf16.Rope.Mixed
   , utf16SplitAt
   , utf16LengthAsPosition
   , utf16SplitAtPosition
+  -- * UTF-8 code units
```

Comment:
Do we need to amend the description of `data Rope` below? 

It's somewhat weird to provide UTF-8 operations from `Data.Text.Utf16` namespace.

Reply:
I've amended the description.

Yeah, perhaps this warrants a module name change? What do you think.

Reply:
What's our full taxonomy now? There are `Data.Text.{Lines,Rope}`, and then `Data.Text.Utf16.{Lines,Rope}` and new `Data.Text.Utf8.{Lines,Rope}`. If `Data.Text.Utf16.Rope.Mixed` now mixes all three variations, how about renaming it to `Data.Text.Mixed.Rope`? And keep `Data.Text.Utf16.Rope.Mixed` for legacy clients, but make it re-export from `Data.Text.Mixed.Rope` only functions it used to provide before.

Reply:
I've gone and done this now.